### DISCUSSION TITLE
Future plans for interaction on multiple levels

### DISCUSSION THREAD

**Discussion by Flor-e** [alkem-io/alkemio#867] [Q&A] [answered]
Currently there is interaction possible on hub, challenge and opportunity level. I find it somewhat confusing sometimes to know on what level I currently am and I feel I am missing out on contributions people have made on 'deeper levels', because I actively have to navigate to a challenge or opportunity. (The hub level is easier accessible because it is only one click away, but still I sometimes stumble upon contributions I had not seen before).

I am curious if there are any thoughts already on how to distinguish better between levels and how to somehow make it easier for people to know where there is activity (without overwhelming them).

**Comment by techsmyth** [comment, marked as answer]
Hi Flori,

Yes there is design work already quite far along to make it clear what level you are at:
- visually: by having the different levels look / feel different
- navigation: making it easier to know where you are

It is an on-going topic, suggest we also bring this up at that backlog review session (still to be planned!).

Cheers, Neil
